# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1247767469 14400
#      Thu Jul 16 14:04:29 2009 -0400
# Node ID 5302df15ad76e9bf9435c846ab3f1edb68a4a7d4
# Parent  3ca88e72f0e89b87900555f8090295ddd2a995a8
__magick_read__.cc: undo unintended change

diff --git a/src/DLD-FUNCTIONS/__magick_read__.cc b/src/DLD-FUNCTIONS/__magick_read__.cc
--- a/src/DLD-FUNCTIONS/__magick_read__.cc
+++ b/src/DLD-FUNCTIONS/__magick_read__.cc
@@ -339,17 +339,17 @@ read_images (const std::vector<Magick::I
 
   retval(0) = im;
 
   return retval;
 }
 
 #endif
 
-DEFUN_DLD (magick_read, args, nargout,
+DEFUN_DLD (__magick_read__, args, nargout,
   "-*- texinfo -*-\n\
 @deftypefn {Function File} {@var{m} =} __magick_read__(@var{fname}, @var{index})\n\
 @deftypefnx{Function File} {[@var{m}, @var{colormap}] =} __magick_read__(@var{fname}, @var{index})\n\
 @deftypefnx{Function File} {[@var{m}, @var{colormap}, @var{alpha}] =} __magick_read__(@var{fname}, @var{index})\n\
 Read images with ImageMagick++.  In general you should not be using this function.\n\
 Instead you should use @code{imread}.\n\
 @seealso{imread}\n\
 @end deftypefn")
@@ -748,17 +748,17 @@ write_image (const std::string& filename
   catch (Magick::Exception& e)
     {
       error ("Magick++ exception: %s", e.what ());
     }
 }
 
 #endif
 
-DEFUN_DLD (magick_write, args, ,
+DEFUN_DLD (__magick_write__, args, ,
   "-*- texinfo -*-\n\
 @deftypefn {Function File} {} __magick_write__(@var{fname}, @var{fmt}, @var{img})\n\
 @deftypefnx {Function File} {} __magick_write__(@var{fname}, @var{fmt}, @var{img}, @var{map})\n\
 Write images with ImageMagick++.  In general you should not be using this function.\n\
 Instead you should use @code{imwrite}.\n\
 @seealso{imread}\n\
 @end deftypefn")
 {
@@ -877,17 +877,17 @@ magick_to_octave_value (const Magick::Im
       st.assign (OUTNAME, magick_to_octave_value (im.NAME ())); \
     } \
   catch (Magick::Warning& w) \
     { \
     }
 
 #endif
 
-DEFUN_DLD (magick_finfo, args, ,
+DEFUN_DLD (__magick_finfo__, args, ,
   "-*- texinfo -*-\n\
 @deftypefn {Loadable File} {} __magick_finfo__(@var{fname})\n\
 Read image information with GraphicsMagick++.  In general you should\n\
 not be using this function.  Instead you should use @code{imfinfo}.\n\
 @seealso{imfinfo, imread}\n\
 @end deftypefn")
 {
   octave_value_list output;
